styles/angular-patternfly.css

```diff
@@ -172,6 +172,43 @@
   padding-right: 5px;
 }
 
+.data-list-pf .list-menu {
+  bottom: 0px;
+  position: absolute;
+  right: -20px;
+  top: 0px;
+}
+
+.data-list-pf .list-menu .btn-link {
+  color: #222222;
+  font-size: 16px;
+  line-height: 1;
+  padding: 4px 0;
+}
+
+.data-list-pf .list-menu .btn-link:hover {
+  color: #0099d3;
+}
+
+.data-list-pf .list-menu.disabled .btn-link,
+.data-list-pf .list-menu.disabled .btn-link:hover {
+  color: #999999;
+  cursor: not-allowed;
```

Comment:
This isn't applied by [`.dropdown-menu > .disabled`](http://getbootstrap.com/components/#dropdowns-disabled)?